code/src/index.css

```diff
@@ -1,13 +1,194 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: Llawysgrifen, source-code-pro, Menlo, Monaco, Consolas,
+    'Courier New', monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: Llawysgrifen, source-code-pro, Menlo, Monaco, Consolas,
+    'Courier New', monospace;
+}
+
+a.active {
+  color: #61dafb;
+}
+h1,
+h2,
+h3,
+p {
+  font-size: 2rem;
+  font-weight: 600;
+  margin-bottom: 10px;
+  color: #fff;
+}
+.movie-poster {
+  position: relative;
+  filter: brightness(100%);
+  transition: filter 0.2s ease-out;
+}
+.movie-poster:hover {
+  filter: brightness(80%);
+}
+
+.movie-container {
+  display: flex;
```

Comment:
There is a gap between the columns in the flexbox. I think you could make the pictures responsive to fill that gap.
We used grid here.